docs/rules/expect-expect.md

```diff
@@ -75,3 +75,23 @@ test('returns sum', () =>
     .run();
 );
 ```
+
+Examples of **correct** code for deep assertion functions with the
+`{ "assertFunctionNames": ["expect", "tester.foo.expect"] }` option:
+
+```js
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "tester.foo.expect"] }] */
+test('nested expect method call', () => {
```

Comment:
I think it would be good to have an example of `SuperTest`, as it's a pretty easy real world example - just taking the first code block from their README will be fine :)

Reply:
Just added an example for SuperTest!